x-pack/plugins/maps/public/bootstrap/get_initial_layers.test.js

```diff
@@ -26,7 +26,7 @@ describe('Saved object has layer list', () => {
       },
     ];
     const layerListJSON = JSON.stringify(layerListFromSavedObject);
-    expect(getInitialLayers(layerListJSON)).toEqual(layerListFromSavedObject);
+    expect(JSON.parse(layerListJSON)).toEqual(layerListFromSavedObject);
```

Comment:
You can just remove this test case since it is really testing JSON.stringify === JSON.parse now that layerListJSON has been removed from the method